src/Backend/Modules/Analytics/Actions/Settings.php

```diff
@@ -160,14 +160,11 @@ private function validateSecretFileForm()
         }
 
         if ($this->form->isCorrect()) {
-            $fileField->moveFile(
-                BACKEND_CACHE_PATH . '/' . $this->getModule() . '/'
-                . $fileField->getFileName()
-            );
+            $authConfigFileContent = \SpoonFile::getContent($fileField->getTempFileName());
```

Comment:
It's not really necessary to put this in a variable, you can just do the getContent in the fork.settings->set method. I'd also use file_get_contents instead of SpoonFile::getContent, since we're trying to remove Spoon from our stack.